### DISCUSSION TITLE
PyDy visualization

### DISCUSSION THREAD

**Discussion by Simone2021space** [sympy/sympy#22622] [Q&A]
Hi,
this question is related to PyDy.
I am using pydy.viz, but scene.display_ipython() does not work for me.

Even in the human_standing_tutorial, I can just visualize an empty panel with buttons.
Is it something related to graphic settings?

![im](https://user-images.githubusercontent.com/89007506/145070566-3f32fd90-859c-4686-875f-bdb0f51f6f75.PNG)

**Comment by moorepants** [comment]
You should open issues/discussion about pydy at the pydy repo.

There are several issues open about this, e.g.: https://github.com/pydy/pydy/issues/388

`.display_ipython()` only works with old versions of ipython widgets.
